.github/workflows/ci.yml

```diff
@@ -177,6 +177,12 @@ jobs:
       with:
         name: ${{ steps.vcpkg-info.outputs.triplet }}-vcpkg-arrow-logs
         path: ${{ steps.vcpkg-info.outputs.root }}/buildtrees/arrow/*.log
+    - name: Dump vcpkg arrow abi info (Unix)
+      if: runner.os == 'Linux' || runner.os == 'macOS'
+      run: cat ./build/${{ steps.vcpkg-info.outputs.triplet }}-release/vcpkg_installed/${{ steps.vcpkg-info.outputs.triplet }}/share/arrow/vcpkg_abi_info.txt
+    - name: Dump vcpkg arrow abi info (Windows)
+      if: runner.os == 'Windows'
+      run: Get-Content ./build/${{ steps.vcpkg-info.outputs.triplet }}/vcpkg_installed/${{ steps.vcpkg-info.outputs.triplet }}/share/arrow/vcpkg_abi_info.txt
```

Comment:
nit: I think it is possible to use powershell or unix tools on all OSes, so there is no need to do it conditionally. But doing it conditionally is fine too.

Reply:
So I originally just had one job for all OSs, as powershell recognizes  `cat` as an alias for `Get-Content`, but the Windows path is slightly different and doesn't have the extra "-release". I could probably figure out how to add that conditionally, but it seemed easier just to have separate jobs.